File: root/programs/VB/Samples/WebApp_sample/ProjectX_sample/Aspx/Common/sampleControl.ascx.vb
'**********************************************************************************
'* フレームワーク・テスト UI（Ｐ層）
'**********************************************************************************

' テスト画面なので、必要に応じて流用 or 削除して下さい。

'**********************************************************************************
'* クラス名        ：WebUserControl
'* クラス日本語名  ：WebUserControl上のEvent Handlerをハンドルする。
'*
'* 作成日時        ：－
'* 作成者          ：生技
'* 更新履歴        ：
'*
'*  日時        更新者            内容
'*  ----------  ----------------  -------------------------------------------------
'*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
'**********************************************************************************

Imports Touryo.Infrastructure.Framework.Presentation

Namespace Aspx.Common
    ''' <summary>WebUserControl class</summary>
    Partial Public Class WebUserControl
        Inherits System.Web.UI.UserControl
        ''' <summary>User ControlにEvent Handlerを実装可能にしたのでそのテスト。</summary>
        ''' <param name="fxEventArgs">Event Handlerの共通引数</param>
        ''' <returns>URL</returns>
        Protected Function UOC_btnUCButton_Click(fxEventArgs As FxEventArgs) As String
            Me.lblResult.Text = "UOC_btnUCButton_Clickを実行できた。"
            Return ""
        End Function
    End Class
End Namespace

File: root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Aspx/Common/sampleControl.ascx.cs
//**********************************************************************************
//* クラス名        ：WebUserControl
//* クラス日本語名  ：WebUserControl上のイベントハンドラをハンドルする。
//*
//* 作成日時        ：－
//* 作成者          ：sas 生技
//* 更新履歴        ：
//*
//*  日時        更新者            内容
//*  ----------  ----------------  -------------------------------------------------
//*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
//**********************************************************************************

// OpenTouryo
using Touryo.Infrastructure.Framework.Presentation;

namespace ProjectX_sample.Aspx.Common
{
    /// <summary>WebUserControl class</summary>
    public partial class WebUserControl : System.Web.UI.UserControl
    {
        /// <summary>ユーザコントロールにイベントハンドラを実装可能にしたのでそのテスト。</summary>
        /// <param name="fxEventArgs">イベントハンドラの共通引数</param>
        /// <returns>URL</returns>
        protected string UOC_btnUCButton_Click(FxEventArgs fxEventArgs)
        {
            Response.Write("UOC_btnUCButton_Clickを実行できた。");

            return "";
        }
    } 
}
